app/src/main/java/com/mapbox/maps/testapp/examples/SecondaryDisplayPresentationActivity.kt

```diff
@@ -0,0 +1,201 @@
+package com.mapbox.maps.testapp.examples
+
+import android.app.Presentation
+import android.content.Context
+import android.content.DialogInterface
+import android.hardware.display.DisplayManager
+import android.os.Bundle
+import android.util.Log
+import android.view.Display
+import android.view.LayoutInflater
+import android.view.View
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.children
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LifecycleRegistry
+import androidx.lifecycle.ViewTreeLifecycleOwner
+import com.mapbox.maps.MapView
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.databinding.ActivitySecondaryDisplayPresentationBinding
+import com.mapbox.maps.testapp.databinding.ItemDisplayInfoBinding
+
+private const val TAG = "SecondaryDisplayPresent"
+
+private const val DISPLAY_INFO_TAG = "DisplayInfo"
+
+class SecondaryDisplayPresentationActivity : AppCompatActivity() {
+  private lateinit var displayManager: DisplayManager
+  private var currentPresentation: Presentation? = null
+  private lateinit var binding: ActivitySecondaryDisplayPresentationBinding
+  private val onDismissListener = DialogInterface.OnDismissListener { dialog ->
+    if (dialog == currentPresentation) {
+      Log.i(TAG, "Presentation was dismissed.")
+      currentPresentation = null
+      updateDisplaysInfo()
+    }
+  }
+  private val displayListener = object : DisplayManager.DisplayListener {
+    override fun onDisplayAdded(displayId: Int) {
+      Log.d(TAG, "Display #$displayId added.")
```

Comment:
debug logs will not be shown in release - perhaps use info?
```suggestion
      Log.i(TAG, "Display #$displayId added.")
```

Reply:
AFAIK, they're shown in release since we don't check if the tag is enabled or not.

Reply:
ah, you're using Android logger and not Common SDK one

Reply:
Yes, I thought it would be better since this is an example that users might copy.